pkgs/os-specific/linux/numworks-udev-rules/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "numworks-udev-rules";
+  version = "unstable-2020-08-31";
+
+  udevRules = fetchurl {
+    # let's pin the latest commit in the repo which touched the udev rules file
+    url = "https://workshop.numworks.com/files/drivers/linux/50-numworks-calculator.rules";
```

Comment:
Is that URL stable? If they touch the file, the hash becomes unstable. Since this is just a two-line file, maybe it's better two just echo these two lines to a file in `installPhase` or perhaps just include the file in nixpkgs? (We do this already in `pkgs/os-specific/linux/wooting-udev-rules`.)

Reply:
Hosted the file in the derivation folder. It'll be updated with the update.sh script.